### ISSUE TITLE
[4.x]: New naming convention for user permissions is very unclear

### ISSUE THREAD

**Issue by darylknight** [craftcms/cms#13370]
I'm setting up more user groups for a Craft 4 site, and I've just seen that the labels for permissions have been changed. In Craft 3, you could set up a user group which allowed a user to create entries and save changes, but they couldn't make those changes live and required a user with the "Publish live changes" permission to actually save the entry.

I find the new naming convention for the ability to "Save entries" very unclear. Please consider changing it back to how it used to be, "Publish live changes".

- "Save Entries" is something that everyone is always able to do, because if they can Edit an entry, then they save their changes, even if the thing they save is turned into a Draft. It's not clear from "Save Entries" that they unable to "save them and make their saved changes live"
- "Publish live changes" is clearer that you can edit an entry and save your changes, but you can't make them live.

Old: 
<img width="372" alt="Old entry permissions" src="https://github.com/craftcms/cms/assets/1250017/424e76bf-3c33-4299-871f-7a8804524e16">

New:
<img width="260" alt="New entry permissions" src="https://github.com/craftcms/cms/assets/47792/5a389a74-ece6-4271-b051-e92db2baf46d">

### Craft CMS version

4.4.14

**Comment by brandonkelly** [comment]
The permission changes were made via #10467. Maybe this helps clear it up?

> (There’s no longer a distinction between saving an enabled or disabled entry, which is what “publish” used to refer to here. Nowadays if a user wants to make a non-live change to an element, they can create a draft.)

**Comment by darylknight** [comment]
I guess closing it means it won't be changed, but no - I still think the wording is confusing. When you're creating new user groups, I think it's now very unclear how you can set up a group that is able to edit, but not publish their changes.

**Comment by MoritzLost** [comment]
@brandonkelly That sounds great!

> Sortof. “Publish” implies publishing to the front-end, which only happens if the entry is enabled. Which is why we stopped using that word in Craft 4 – since there’s no longer a differentiation between saving disabled entries and enabled entries.

Good to know the reasoning! But is it important to have that distinction in the wording of the permission? After all, users with that permission can always enable a disabled entry. And in some cases, disabled entries could still be visible in the frontend. What really matters here, I think, is that users can update/create the live/published/canonical version of a published entry (technically not _canonical_ since unpublished drafts are also canonical). In the end, the permission allows users to update/create a version of an entry that _may_ be visible in the frontend – in that sense, I still think _publish_ is a better wording than _save_.

> But now that I think about it, it’s a little awkward that we are differentiating between creating drafts of existing entries and creating drafts of new entries. So maybe it makes more sense for us to just combine those into a single “Create drafts” permission:

I think that makes perfect sense! That stood out to me as well when I wrote my comment above, though I didn't want to suggest this change since I wasn't sure if it would be a BC break 😅 It might be, in case some frontend code relies on users with the _View_ permission to be able to create drafts for existing entries.

In the meantime, I think the wording changes you mentioned are a solid improvement 👍 Though I would still suggest a change to the _Save_ permission, that one is the least clear in my opinion.

**Comment by MoritzLost** [comment]
@brandonkelly Absolutely! Though I would still argue that _Save_ is more confusing than _Publish_, even given the nuances of what _Publish_ might mean, since _Save_ can also mean a bunch of things (like saving drafts).

Maybe there's another wording that better captures the difference between creating drafts and creating something that the public might see?

**Comment by mmikkel** [comment]
> Create entries → Create drafts

This would be a very good call IMO, especially considering the editing UI uses "Create entry" for when a draft is first make into a real entry (an action that actually requires the "Save entries" permission).

> “Publish” implies publishing to the front-end, which only happens if the entry is enabled.
> I would still argue that Save is more confusing than Publish

"Save entries" is a bit fuzzy because it's technically required for (what is often perceived to be) a few different actions, like applying drafts, saving existing entries, and creating new entries from drafts. It's probably going to be difficult to find a name that accurately describes everything it does, though.

Maybe instead of renaming the permission, it could be given an info toggle (similar to the one a few of the user permissions have), explaining in a bit more detail what the permission does?

![CleanShot 2023-06-30 at 18 00 23](https://github.com/craftcms/cms/assets/298510/b52bfe0a-c045-40e7-ba64-fa8022b55d50)
